### ISSUE TITLE
Third-party python packages completion in virtualenv

### ISSUE THREAD

**Issue by cinuor** [roxma/nvim-completion-manager#113]
I find that NCM does not support third-party packages when it's in virtualenv.

Environment：
Ubuntu 16.04

Installed Plugins：
Plug 'roxma/nvim-completion-manager'
Plug 'Shougo/neosnippet'
Plug 'Shougo/neosnippet-snippets'

Installed Packages:
neovim,jedi (both in system python2 and python3)

1.It works fine and show third-party packages when using python3 in system.

2.When I create a virtualenv(python3),getting into it and install `neovim` ,`jedi`.
It works fine,and show third-party packages in virtualenv

3.When I create a virtualenv(python2), install `neovim` ,`jedi` and get into it.
It does not show third-party packages in virtualenv, only show thoes in system python3 environment.

In short, **NCM does not support third-party completion in python2 virtualenv now**.

Thanks.

**Comment by roxma** [comment]
Please test the newest version of NCM.

**Comment by cinuor** [comment]
It works ! Thanks!
